mappings/net/minecraft/block/BlockAir.mapping

```diff
@@ -18,3 +18,5 @@ CLASS ana net/minecraft/block/BlockAir
 		ARG 0 state
 	METHOD b isFullBoundsCubeForCulling (Lave;)Z
 		ARG 0 state
+	METHOD c (Lave;)Z
```

Comment:
This is used on a lot of the blocks, might be best to see if you can map this one.